core/formats/skip_list.cpp

```diff
@@ -45,8 +45,7 @@ namespace irs {
 void SkipWriter::Prepare(
   size_t max_levels, size_t count,
   const memory_allocator& alloc /* = memory_allocator::global() */) {
-  max_levels_ =
-    std::clamp(max_levels, size_t{1}, ::max_levels(skip_0_, skip_n_, count));
```

Comment:
Why is it UB?

Reply:
`::max_levels(skip_0_, skip_n_, count)`  can return 0

Reply:
👍